### PR TITLE
Stata models

### PR DESCRIPTION
**Pull request by josephignace**
Prepared scripts for Stata models.
Models to be run using Stata needs to be updated
To test the models use test prevax-vascular-dementia.
Lines 81-85 removed (exclusive to gastro repo).

### CODE REVIEW

**Review by venexia — CHANGES_REQUESTED** [review]
Hi Jose. Thank you for providing the action name so I could test this locally. I have confirmed the code works. However, the Stata code is expecting three inputs (cpf, day0, extf) and you have only specified one in your action - please can you resolve this? Also, there is an optional change, which is to make a single model output file for both R and Stata results.

**Review by venexia — CHANGES_REQUESTED** [review]
Hi Jose. Thank you for addressing the comments. One further change is required. The input for the argument `day0` is `TRUE`/`FALSE` depending on whether you want an estimate for day zero (`TRUE`) or not (`FALSE`). Currently, you have set the parameter to be the name - for instance, for the action 'stata_cox_ipw-cohort_prevax-main-vascular_dementia', the argument `day0` is set to `cohort_prevax-main-vascular_dementia`.

**Comment by josephignace** [comment]
Day0 argument added (following MH approach).
Files names changed in the cox_model.do ("ready-").
Simplified list of models to be run using Stata and Stata output added (create project actions) to test the analysis locally (this will be changed to the full list once the PR is ready).

**Review by venexia — APPROVED** [review]
Looks good. Thanks, Jose.

File: analysis/stata/cox_model.do [hunk 1]
```diff
@@ -233,4 +232,4 @@ egen events = count(patient_id), by(term)
 keep term median_tte events
 duplicates drop
 
-export delimited using "output/`name'_median_fup.csv", replace
+export delimited using "output/ready-`name'_median_fup.csv", replace
```
[the remark this answers is not in the document]
Comment (josephignace) [reply analysis/stata/cox_model.do]: I made the change, both cox_model.do and create_project_actions.

**Comment by josephignace** [comment]
@venexia Should I merge this PR? Or should I wait for the output release to update the list of models to be run using Stata (I would prefer this option)?

**Pull request merged**